doc/tutorials/workshop/content/exercises/11-node-rules.md

```diff
@@ -0,0 +1,36 @@
+---
+Title: Node Rules
+PrevPage: 10-rule-parametrization
+NextPage: 12-common-rules
+---
+
+Node Rules
+===================
+
+## Platform vs Node rules
+
+One important distinction to make is whether the configuration checked is
+a Kubernetes resource (a Platform check) or a setting in the node of the
+cluster (a Node check).
+
+The Platform rules check the Kubernetes API Resoruces.
+While the Node rules check configuration of the node's operating system.
+
+So far the rules we created were Platform rules, they checked a Kubernetes
+configuration. But one can also check configuarations of the operating
+system at the node level, at that is what we'll be doing in this section.
+
+### Selecting the nodes to check
+
+The Node rules can be assigned to check all nodes or only master nodes.
+
+To set a node check to run on all nodes, set the rule's platform to:
+`platform: {{{ product }}}-node`
+
+To set a node check to run on only on master nodes, set the rule's platform to:
+`platform: {{{ product }}}-master-node`
+
```

Comment:
should we tell the user that this is different from setting the role in ScanSetting? And if a master rule is being ran on a worker node, it will have not-applicable result

Reply:
Hmm, good point. I think I'd keep it separate for now, so that we don't overload the learner with too much information.